src/Microsoft.Framework.Runtime.Roslyn.Interfaces/IAfterCompileContext.cs

```diff
@@ -21,5 +22,7 @@ public interface IAfterCompileContext
         Stream XmlDocStream { get; set; }
 
         IList<Diagnostic> Diagnostics { get; }
+
+        IEnumerable<IMetadataReference> References { get; }
```

Comment:
Do you want it immutable like in Roslyn? Not sure of what's your convention in this project.